samples/hello/hello.py

```diff
@@ -29,71 +27,43 @@
 from opensearch_sdk_py.transport.tcp_header import TcpHeader
 
 
-async def handle_connection(conn: Any, loop: asyncio.AbstractEventLoop) -> None:
-    try:
-        conn.setblocking(False)
+class HelloExtensionHost(AsyncHost):
```

Comment:
Why does this part need to stay in the extension?  It's essentially boilerplate (or will be once we do response handling properly).

Reply:
I think this goes into `AsyncExtensionHost` that can `.execute(extension)` and that implements extension-specific handling (request/response loop) on top of `AsyncHost` that implements the generic TCP/IP request/response.